### ISSUE TITLE
Getting Started - NuGet versions / DllLoadException

### ISSUE THREAD

**Issue by govert** [kaby76/Campy#2]
Your project looks cool, but we're struggling to get things running.

For the Visual Studio project we get errors finding the swigged.llvm package version 70.0.0 (which one of your comments indicated have not been published yet).

For the command line / bash instructions we compile successfully, but then get an error:
* "`DllLoadException` - Unable to load DLL campy-runtime-wrapper", even though the .dll is present in the 'publish' directory.

We're running VS 2017 under Windows 10.

Can you give any advise on how we might try to get a running version?
Do we also have to compile and package swigged.llvm and publish to a local dir, or can you suggest a simpler route?

**Comment by kaby76** [comment]
Right now Campy master branch is pretty much in flux, I still haven't finished the v0.0.14 release. I haven't checked in the stuff for the LLVM library because I haven't built it yet for Linux since I too am working on Windows. I'll update Swigged.LLVM, as well as the newer CUDA library, for Windows, without the Linux build.

**Comment by govert** [comment]
@kaby76 Thank you for the quick reply! We'll look out for a Windows build that we can try.

**Comment by kaby76** [comment]
Thanks for pointing out this problem! I've reproduced it, and I'll have a fix in the next release this week. I'll send you email when it's done.

**Comment by kaby76** [comment]
I've made a release of Campy (0.0.14) and it should fix your problem. The issue was a version skew in building the code. It now is a clean build, and with a much improved compiler in this release. CUDA GPU Toolkit 9.2.148 is required, and won't work with anything else. Any further problems, send me email at ken.domino@gmail.com. Thanks!

**Comment by jonathan3698** [comment]
I first tried to install through the package manager, but I was getting errors. The path was not correct (it was C:\Users\Kayne something, and I didn't know how to fix this, so I just downloaded the whole project on Github.

Then I get "BadImageFormatException", so I tried changing it to 64-bits, because why not? And suddenly this error disappeared. Then I get the error "Key is already added" for a dictionary, so I commented those out and then finally, I am getting the same error... I spent like 6 hours to try to fix the problem, but no matter where I put the DLL, it just doesn't find it :(

This library seems awesome but it's a pain in the ass to make it work. However I think it has a really great potential.

Any idea on what should I do kaby?

Thank you in advance...
